lib/tools/adb-commands.js

```diff
@@ -1608,4 +1609,26 @@ methods.screenrecord = function (destination, options = {}) {
   return new SubProcess(this.executable.path, fullCmd);
 };
 
+/**
+ * Performs the given editor action on the focused input field.
+ * This method requires Appium Settings helper to be installed on the device.
+ * No exception is thrown if there was a failure while performing the action.
+ * You must investigate the logcat output if something did not work as expected.
+ *
+ * @param {string|number} action - Either action code or name. The following action
+ *                                 names are supported: `normal, unspecified, none,
+ *                                 go, search, send, next, done, previous`
+ */
+methods.performEditorAction = async function (action) {
+  log.debug(`Performing editor action: ${action}`);
+  const defaultIME = await this.defaultIME();
+  await this.enableIME(APPIUM_IME);
+  try {
+    await this.setIME(APPIUM_IME);
+    await this.shell('input', 'text', `/${action}/`);
```

Comment:
`[]` is needed
`await this.shell(['input', 'text', `/${action}/`]);`

Reply:
good catch. thanks